include/elements/3d/tetrahedron_quadrature.tcc

```diff
@@ -0,0 +1,54 @@
+// Getting the quadratures for Tnquadratures = 1
+template <>
+inline Eigen::MatrixXd mpm::TetrahedronQuadrature<3, 1>::quadratures() const {
+  Eigen::Matrix<double, 3, 1> quadratures;
+  quadratures(0, 0) = 1.0 / 4;
+  quadratures(1, 0) = 1.0 / 4;
+  quadratures(2, 0) = 1.0 / 4;
+
+  return quadratures;
+}
+
+// Getting the weights for Tnquadratures = 1
+template <>
+inline Eigen::VectorXd mpm::TetrahedronQuadrature<3, 1>::weights() const {
+  Eigen::VectorXd weights(1);
+  weights(0) = 1.0 / 6.;
+
+  return weights;
+}
+
+// Getting the quadratures for Tnquadratures = 4
+template <>
+inline Eigen::MatrixXd mpm::TetrahedronQuadrature<3, 4>::quadratures() const {
+  Eigen::Matrix<double, 3, 4> quadratures;
```

Comment:
do we want to try to find a source to put here? I'm sure the values are fine, but I have no clue where they come form

Reply:
A good idea! Maybe in the header file.